### DISCUSSION TITLE
igorpecovnik is rude and useless in replays

### DISCUSSION THREAD

**Discussion by easy-and-simple** [community/community#56046] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Select Feature Area

- [X] Issues
- [X] Projects

### Body

igorpecovnik is rude and useless in replays. This guys is everywhere with his rude and useless spam replays. On top of all he have admin/moderator access and blocks my posts.
Why github allows to such rude and useless guys to be rude with users and to block them instead to discuss resolve the issues.
This guy replays to any support request with his rude and useless messages. It is not mandatory to replay if cant help and can't say anything useful, but he replays and dont miss any post. 
Typical replay of igorpecovnik after insulting people is: go and read it at https://docs.armbian.com/
This is main documentation page, and not documentation for specific issue.
Any bot will make much useful replay than igorpecovnik
I think there must be some mechanism for controlling harmful behaviour of igorpecovnik and other people with moderator/admin rights
Project armbian that is maintained by igorpecovnik asks people for their usernames and passwords when they use armbian build tool, but after username and password are entered nothing happens, so I suspect igorpecovnik  steals peoples credentials for github

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment, marked as answer]
Hi @easy-and-simple 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
